src/ssort/_utils.py

```diff
@@ -48,6 +49,18 @@ def __call__(self, arg: Any, *args: Any) -> _T:
 single_dispatch = _SingleDispatch
 
 
+def escape_path(path):
+    """
+    Takes a `pathlib.Path` object and returns a string representation that can
+    be safely copied into the system shell.
+    """
+    if sys.platform == "win32":
+        # TODO
+        return str(path)
+    else:
+        return shlex.quote(str(path))
```

Comment:
I don't think this is correct. We are not passing these paths to a shell so quoting them changes their meaning.

Suppose a file with a space in it's path, `hello world.py` for example, exists. The following will fail:
```python
open(shlex.quote("hello world.py"))
```

With this traceback:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
FileNotFoundError: [Errno 2] No such file or directory: "'hello world.py'"
```

You see, the `open` function, like all other functions which operate on paths, does not understand that the single quotes are not part of the literal path.

This does, however, seem to demonstrate that we lack tests on file paths with complex symbols (spaces, quotes, etc.).